UPGRADE.md

```diff
@@ -19,6 +19,86 @@ doctrine:
                 is_bundle: false
 ```
 
+### Field Descriptor interface changed
+
+The field descriptor parameter `$default` and `$disabled` where combined into a new parameter `$visibility`:
+Use the following table for upgrading:
+
+| Disabled | Default  | Visiblity
+|----------|----------|--------------
+| false    | true     | FieldDescriptorInterface::VISIBILITY_ALWAYS (always)
+| false    | false    | FieldDescriptorInterface::VISIBILITY_NEVER (never)
+| true     | true     | FieldDescriptorInterface::VISIBILITY_YES (yes)
+| true     | false    | FieldDescriptorInterface::VISIBILITY_NO (no)
```

Comment:
this list was false and was fixed in: https://github.com/sulu/sulu/pull/3991/files